### PR TITLE
Ome redesign master

### PR DESCRIPTION
**Pull request by lunson**
Current design work in progress

### CODE REVIEW

**Comment by snoopycrimecop** [comment]
Empty PR description. Please add a short summary of the PR scope and some testing instructions.

**Comment by hflynn** [comment]
Added description

**Comment by hflynn** [comment]
Staging site updated

**Comment by hflynn** [comment]
Updating staging site

File: _layouts/omero-features.html [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+        <header class="header">            
+            <h1 class="headline">{{page.feature}} with OMERO</h1>            
+            <ul class="header-subnav">
+                <li><a href="{{ site.baseurl }}/products/omero/"><i class="fa fa-arrow-left"></i> Overview</a></li>
+                <li><a href="#">NEW???</a></li>
+                <li><a href="{{ site.baseurl }}/products/omero/import/">Import</a></li>            
+                <li><a href="{{ site.baseurl }}/products/omero/organize/">Organize</a></li>            
+                <li><a href="{{ site.baseurl }}/products/omero/view/">View</a></li>            
+                <li><a href="{{ site.baseurl }}/products/omero/analyze/">Analyze</a></li> 
+                <li><a href="#">Publish???</a></li>
```
Comment (hflynn) [inline _layouts/omero-features.html]: Yes, we have https://www.openmicroscopy.org/site/products/omero/omero-platform-v4#publish to mine for content

File: products/omero/index.html [hunk 2]
```diff
@@ -29,89 +29,74 @@ <h5 class="subheader">With our dedicated user help website and extensive technic
...
+                    export as OME-TIFF or movie formats or quickly create
+                    figures for publication.
+                </p>
+                <a href="{{ site.baseurl }}/products/omero/export/" class="button expanded">Learn More</a>
+            </div>
+            <div class="column">
+                <a href="#">
+                    <img class="thumbnail" src="http://placehold.it/300x150">
+                </a>
+                <h5 class="text-center">Publish? Utility Functions?</h5>
```
Comment (hflynn) [inline products/omero/index.html]: Publish
Publish your raw data straight from the server using URLs to link directly to the web-based image viewer in a manuscript or embed images into your website.

**Comment by hflynn** [comment]
https://snoopycrimecop.github.io/www.openmicroscopy.org/products/omero/ shows the new layout for the features

**Comment by jburel** [comment]
Discussed the new layout with @hflynn today, it allows the ability to follow a "trail" e.g. import and switch to another one e.g. organize without having to go back to the entry point. I think it works

Looking at the "group" specific page e.g. scientists:

* in the overview section, we will need to review some of the text (outside the scope of this PR) but it will be good to add  a link under each point of text to point to the corresponding "topic" page.
e.g. under ``Upload your data via the desktop app or command line, use our custom dropbox tool, or have your facility manager import it for you`` I will add  link to ``products/omero/import/``
* maybe a go back arrow similar to the one on the products page for consistency

**Comment by hflynn** [comment]
@jburel Liza isn't going to be working on this again until next week so I suggest we merge this and I'll do the further work on porting the rest of the features into the new structure on a new PR rather than adding more to this one - okay?

**Comment by jburel** [comment]
@hflynn makes sense better to have smaller PRs to adjust the comments
merging

**Pull request merged**